src/backend/crossterm.rs

```diff
@@ -81,21 +67,18 @@ where
     W: Write,
 {
     fn clear(&mut self) -> io::Result<()> {
-        queue!(self.stdout, Clear(ClearType::All));
-        self.stdout.flush();
-        Ok(())
+        queue!(self.stdout, Clear(ClearType::All))?;
+        self.stdout.flush()
```

Comment:
Although this is good, take notice that when you flush the screen, a system call is made to flush the output buffer to the terminal screen. Queue prevents this behavior and make's you able to handle flushing your self. That might be useful in the case when you call clear 'a lot of times'. Crossterm also supports `execute` this willl execute the command for you by calling flush. 

Instead of this
```rust
        queue!(self.stdout, Clear(ClearType::All))?;
        self.stdout.flush()
```

you can do this:

```rust 
        execute!(self.stdout, Clear(ClearType::All))?;
```